### DISCUSSION TITLE
ddns-updater / pi-hole

### DISCUSSION THREAD

**Discussion by shayfer** [qdm12/ddns-updater#329] [General]
ddns-updater is working great for me with Google Domains and Docker/Portainer on Ubuntu.
I noticed an odd issue is when I use ddns-updater and pi-hole. It shows the container status as unhealthy but that is not true. It still updates with no issues and when I go to the portal page it shows everything as OK. When I changed the DNS server on the ddns-updater container back to my router is shows healthy. I looked through the Query logs in pi-hole to see what might be blocking it but do not see anything. I am not sure if it's a pi-hole bug or something with ddns-updater. Not a big deal just curious if anyone else noticed this. 
Thanks!

**Comment by qdm12** [comment]
What do you get in your logs? This can happen for example if you use a public hostname (i.e. domain.com) for an internal IP address

**Reply by shayfer** [reply]
Thanks for the reply! I do not have the logs anymore. I discovered my pfsense server could do ddns updates. I'll close this.
